app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split()
+    if len(command_ls) == 3:
+        prefix = command.split()[0]
+        file_name = command.split()[1]
+        copy_file_name = command.split()[2]
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()